### DISCUSSION TITLE
Vontar X2: ./aml-multiboot-setup.sh not working

### DISCUSSION THREAD

**Discussion by razvanelgsm** [devmfc/debian-on-amlogic#57] [Q&A] [answered]
Little help please

I want to install on emmc so I run ./aml-multiboot-setup.sh with no error but after reboot 
and run ./aml-install-to-emmc.sh I get: Error No multiboot detected. You should setup multiboot first. Run ./aml-multiboot-setup.sh and reboot.
I run again ./aml-multiboot-setup.sh and check dmesg and have this error:

[ 2250.660727] blk_print_req_error: 1 callbacks suppressed
[ 2250.660748] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.663012] buffer_io_error: 166 callbacks suppressed
[ 2250.663021] Buffer I/O error on dev mmcblk1, logical block 29696, lost async page write
[ 2250.665014] Buffer I/O error on dev mmcblk1, logical block 29697, lost async page write
[ 2250.666117] Buffer I/O error on dev mmcblk1, logical block 29698, lost async page write
[ 2250.667118] Buffer I/O error on dev mmcblk1, logical block 29699, lost async page write
[ 2250.668112] Buffer I/O error on dev mmcblk1, logical block 29700, lost async page write
[ 2250.669144] Buffer I/O error on dev mmcblk1, logical block 29701, lost async page write
[ 2250.670146] Buffer I/O error on dev mmcblk1, logical block 29702, lost async page write
[ 2250.671144] Buffer I/O error on dev mmcblk1, logical block 29703, lost async page write
[ 2250.672141] Buffer I/O error on dev mmcblk1, logical block 29704, lost async page write
[ 2250.673173] Buffer I/O error on dev mmcblk1, logical block 29705, lost async page write
[ 2250.686650] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.701706] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.713302] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.739984] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.756633] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.767678] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.783575] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.795931] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
[ 2250.821385] I/O error, dev mmcblk1, sector 237568 op 0x1:(WRITE) flags 0x800 phys_seg 16 prio class 2
tvbox:~#

Please help, thx!

**Comment by Eng3lS** [comment]
Very similar to mmc overfrequency. As here https://github.com/devmfc/debian-on-amlogic/discussions/20#discussioncomment-9069887

**Reply by razvanelgsm** [reply]
Thanks, I already look all the discussions on this problem but I don't manage yet to solve it.

**Comment by razvanelgsm** [comment, marked as answer]
Solved with 20 Mhz, here is the log installing on emmc:

Partition Table: msdos
Disk Flags:

Number  Start   End     Size    Type     File system  Flags
 1      742MB   1279MB  537MB   primary               lba
 2      1279MB  31.3GB  30.0GB  primary

Installing missing package [ dosfstools ]
Selecting previously unselected package dosfstools.
(Reading database ... 14377 files and directories currently installed.)
Preparing to unpack .../dosfstools_4.2-1_arm64.deb ...
Unpacking dosfstools (4.2-1) ...
Setting up dosfstools (4.2-1) ...
mkfs.fat 4.2 (2021-01-31)
mke2fs 1.47.0 (5-Feb-2023)
Discarding device blocks: done
Creating filesystem with 7321600 4k blocks and 1831424 inodes
Filesystem UUID: 9dcc88a6-b771-4817-ae80-bcc1c3de8ada
Superblock backups stored on blocks:
        32768, 98304, 163840, 229376, 294912, 819200, 884736, 1605632, 2654208,
        4096000

Allocating group tables: done
Writing inode tables: done
Creating journal (32768 blocks): done
Writing superblocks and filesystem accounting information: done

[ done ]
 Mounting newly created filesystems... [ done ]
Copy /boot to boot partition... [ done ]
Creating rootfs... [ done ]
Copying rootfs files...
Installing missing package [ rsync ]
Selecting previously unselected package rsync.
(Reading database ... 14412 files and directories currently installed.)
Preparing to unpack .../rsync_3.2.7-1_arm64.deb ...
Unpacking rsync (3.2.7-1) ...
Setting up rsync (3.2.7-1) ...
rsync.service is a disabled or a static unit, not starting it.
Copying [ /etc ]
      1,250,072  98%    2.50MB/s    0:00:00 (xfr#259, to-chk=0/839)
Copying [ /var ]
    190,692,819  99%   55.27MB/s    0:00:03 (xfr#1320, to-chk=0/1391)
Copying [ /home ]
              0 100%    0.00kB/s    0:00:00 (xfr#0, to-chk=0/1)
Copying [ /usr ]
    467,128,598  99%   16.24MB/s    0:00:27 (xfr#11272, to-chk=0/13797)
Copying [ /lib ]
              0   0%    0.00kB/s    0:00:00 (xfr#0, to-chk=0/1)
Copying [ /boot ]
     36,935,511 100%   71.97MB/s    0:00:00 (xfr#309, to-chk=0/345)
Copying [ /opt ]
              0 100%    0.00kB/s    0:00:00 (xfr#0, to-chk=0/1)
Copying [ /srv ]
              0 100%    0.00kB/s    0:00:00 (xfr#0, to-chk=0/1)
Copying [ /bin ]
              0   0%    0.00kB/s    0:00:00 (xfr#0, to-chk=0/1)
Copying [ /root ]
      4,248,953 100%   68.63MB/s    0:00:00 (xfr#13, to-chk=0/18)
Copying [ /sbin ]
              0   0%    0.00kB/s    0:00:00 (xfr#0, to-chk=0/1)
[ done ]
Creating new fstab... [ done ]
Updating boot.config... [ done ]
Wait for sync... [ done ]
Cleaning up...  [done]
